packages/react-composites/tests/browser/call/hermetic/VideoGallery.test.ts

```diff
@@ -110,14 +112,26 @@ test.describe('VideoGallery tests', async () => {
     await page.goto(buildUrlWithMockAdapter(serverUrl, initialState));
 
     // pin remote video tiles up to the max allowed in the call composite
-    for (let i = 0; i < 4; i++) {
-      // click the last 'more options' button on the page which is presumably on an unpinned remote video tile
-      await pageClick(page, dataUiId('video-tile-more-options-button') + ' >> nth=-1');
+    for (let i = 1; i < 5; i++) {
+      await waitForSelector(page, dataUiId(IDS.videoTile) + ` >> nth=${i}`);
```

Comment:
Great fix here. The ith video tile will always be unpinned because you pin them in order.